src/actions/projectMembers.js

```diff
@@ -92,6 +92,33 @@ export const fetchAllMembers = projectId => {
   };
 };
 
+/**
```

Comment:
Thank you for the explanation of the code on top, make it easy for newbie like me to review PR.